src/modules/users/user_service.ts

```diff
@@ -140,4 +147,52 @@ export class UserService {
 
     return this.userRepository.updateStatus(payload, id)
   }
+
+  public verify = async (request: UserEntity.RequestBodyVerify, id: string) => {
+    const item: any = await this.userRepository.findById(id)
+    if (!item)
+      throw new HttpError(httpStatus.NOT_FOUND, lang.__('error.exists', { entity: 'user', id }))
+
+    if (item.status_partner !== StatusPartner.WAITING)
+      throw new HttpError(httpStatus.BAD_REQUEST, lang.__('error.users.partner.verified'))
+
```

Comment:
mau tanya, best practicenya memang ga perlu pakai kurung kurawal `{}` ya kang @ayocodingit ?

Reply:
Kurung kurawal itu untuk menyisipkan text dalam variable kang ke dalam language,
Jadi kenapa `lang.__('error.users.partner.verified'))` tidak ada param yang kedua karena tidak ada variable yang akan disisipkan

Reply:
hehe maksud saya if nya kang.
```suggestion
    if (!item) {
      throw new HttpError(httpStatus.NOT_FOUND, lang.__('error.exists', { entity: 'user', id }))
    }

    if (item.status_partner !== StatusPartner.WAITING) {
      throw new HttpError(httpStatus.BAD_REQUEST, lang.__('error.users.partner.verified'))
    }
```

Reply:
oh iya kang kalau hanya satu baris gak perlu menggunakan kurung kurawal, kalau beberapa baris harus gunakan `{}`

maaf soalnya tadi agak ambigu mana yang di pertanyakannya hehe 🙏